memberships/views.py

```diff
@@ -16,6 +23,49 @@
 from .forms import *
 from .models import Member, Membership
 from .services import StripeGateway
+from .tokens import email_verification_token
+from .tasks import task_send_email
+
+# def validate_recaptcha(response):
+#     url = 'https://www.google.com/recaptcha/api/siteverify'
+#     secret = settings.RECAPTCHA_SECRET_KEY
+#     payload = {
+#         'secret': secret,
+#         'response': response
+#     }
+#     data = urllib.parse.urlencode(payload).encode("utf-8")
+#     request = urllib.request.Request(url, data=data)
+#     response = urllib.request.urlopen(request)
+#     result = json.loads(response.read().decode())
+#     success = result.get('success')
+#
+#     if (not result.get('success')) or (float(result.get('score')) < 0.5):
+#         return 'fail'
+#
+#     return result
```

Comment:
Last thing I can see is just all this commented code. It would be much clearer if this was not here.